moveit_simple/include/moveit_simple/moveit_simple.h

```diff
@@ -383,6 +412,10 @@ class Robot
   bool isReachable(std::unique_ptr<TrajectoryPoint> & point, double timeout,
                    std::vector<double> joint_seed = std::vector<double>() ) const;
 
+  bool jointTrajectoryCollisionCheck(control_msgs::FollowJointTrajectoryGoal & goal,
+                                     int & collision_points,
```

Comment:
This parameter seems redundant with the return value.  Could you just return `0` if not collisions are found?

Reply:
ok